src/App.css

```diff
@@ -17,27 +17,29 @@
 
 .book-container {
   display: flex;
-  padding: 3%;
-  gap: 30%;
-  border: 1px solid blue;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 4rem;
+  width: 90%;
+  padding: 1%;
+  margin-bottom: 1rem;
+  background-color: white;
+  border: 1px solid rgb(221, 229, 224);
```

Comment:
- [ ] Well done on this project. But if you look closely at the Zeplin design, you will see a huge disparity with your project style implementation. See below the list of things you should fix 👇🏾 

    - Use the correct colors where necessary. inspect the Zeplin design to see,
    - Use the correct font family and font size as used in the Zeplin
    - Ensure unnecessary borders are removed from buttons.
    - The progress bar has a percentage figure beside it and it is not like you have done. compare them. You can use an npm package. 
    - The chapter text is missing 

You will more issues pointed out in the image below.

## Your design
![image](https://github.com/Simpleshaikh1/Book-Store/assets/84156714/90fee2c1-f971-4c0c-8b49-f3f94af1c773)

## Zeplin design
![image](https://github.com/Simpleshaikh1/Book-Store/assets/84156714/688bc8ba-f804-4a00-85d7-d28b0af9b968)

Please work on the styling some more. I have not touched all the issues but you should ensure your project resembles the Zeplin design as much as possible.